src/stores/GasPriceStore.js

```diff
@@ -74,6 +64,31 @@ class GasPriceStore {
         return Promise.resolve()
       })
   }
+
+  @computed
```

Comment:
I converted the descriptions to computed values because functions cannot be serialized and saved in local storage.